skiperator/controllers/application/deployment.go

```diff
@@ -0,0 +1,232 @@
+package applicationcontroller
```

Comment:
Is this a local backup that was committed in by accident?

Reply:
Agreed, it seems this file was added by accident and should be removed before merging. It's still in the branch so I'm unresolving this thread